src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohort.java

```diff
@@ -1,9 +1,7 @@
 package org.broadinstitute.hellbender.tools.gvs.extract;
 
-import htsjdk.variant.vcf.VCFFormatHeaderLine;
-import htsjdk.variant.vcf.VCFHeader;
-import htsjdk.variant.vcf.VCFHeaderLine;
-import htsjdk.variant.vcf.VCFHeaderLineType;
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.variant.vcf.*;
```

Comment:
was this intentional? Are we changing styles?

Reply:
intellij autoformatting -- I didn't even know we had an official style :D but happy to change it back